personal/james/chapter1/README.md

```diff
@@ -0,0 +1,4 @@
+[confluence-links]
+	korean : https://james-heo.atlassian.net/wiki/spaces/LN/pages/33200
+	english : https://james-heo.atlassian.net/wiki/spaces/LN/pages/65545/Why+Terraform
```

Comment:
```suggestion
# confluence-links
- korean : https://james-heo.atlassian.net/wiki/spaces/LN/pages/33200
- english : https://james-heo.atlassian.net/wiki/spaces/LN/pages/65545/Why+Terraform
```

Reply:
Thank you :)

Reply:
If you want to apply my suggestion, Click the `Commit suggestion` button and add a commit  

However, this is just a "suggestion", so you don't have to 😄